packages/clients/meldemichel/src/plugins/AfmButton/AfmButton.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-btn
+    v-if="afmUrl"
+    :target="mapStateReady ? '_blank' : ''"
+    color="primary"
+    class="meldemichel-afm-button"
+    :href="mapStateReady ? afmUrl : '#'"
+    large
+    @click="click"
+    @keydown.space="click"
+  >
+    <v-icon small>fa-map-location</v-icon>
+    {{ $t('common:plugins.meldemichelAfmButton.buttonText') }}
+  </v-btn>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { mapGetters } from 'vuex'
+
+export default Vue.extend({
+  name: 'MeldemichelAfmButton',
```

Comment:
Especially here it is clear, that the name should simply be `AfmButton`.

I was also thinking about naming the file `index.vue`, as the path `.../AfmButton/AfmButton.vue` looks weird

Reply:
Each plugin has a vue name that matches its folder name. Sometimes with a `src` in between, which makes it less obvious. Since we're in Meldemichel/AfmButton, `MeldemichelAfmButton` makes perfect sense as a name.